htdocs/admin/dict.php

```diff
@@ -2221,7 +2221,7 @@ function fieldList($fieldlist, $obj = '', $tabname = '', $context = '')
 
 	foreach ($fieldlist as $field => $value) {
 		if ($value == 'entity') {
-			$withentity = $obj->{$value};
+			$withentity = empty($obj->{$value}) ? '' : $obj->{$value};
```

Comment:
If $obj->{$value} is defined and is 0, the change will have for result
$withentity = '';
So it is a "not compatible" change.
May be a !isset is better.